test/e2e/setup/rosa/README.md

```diff
@@ -0,0 +1,91 @@
+# Setup
+
+## Setup Maestro server and agent
+
+This setup demonstrate how to deploy the Maestro with AWS IoT and AWS RDS PostgreSQL.
+
+1. Run following command to deploy a Maestro server on a given cluster
+
+```sh
+export AWS_IOT_REGION="<your_aws_iot_region>"
+export CLUSTER_REGION="<your_cluster_region>"
+export CLUSTER_VPC="<your_cluster_vpc>"
+export KUBECONFIG="<your_cluster_kubeconfig>"
```

Comment:
No aws credentials provided? use default location?

Reply:
This required a logined aws command

Reply:
Add Prerequisites part to address this